chapters/extend/executor.rst

```diff
@@ -221,11 +257,154 @@ Then simply use it in Jina CLI by specifying ``jina pod --uses=my.yml``, or ``Fl
     If you use customized executor inside a :class:`jina.executors.CompoundExecutor`, then you only need to set ``metas.py_modules`` at the root level, not at the sub-component level.
 
 
-I Want to Contribute it to Jina
--------------------------------
+Customize Executor in Action: CLIP Encoder
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`CLIP <https://github.com/openai/CLIP>`_ (Contrastive Language-Image Pre-Training) is a neural network trained on a variety of (image, text) pairs.
+It can be instructed in natural language to predict the most relevant text snippet given an image.
+
+The pre-trained CLIP model is able to transform both images and text into the same latent space,
+where they can be compared using a similarity measure.
```

Comment:
the `they` is not clear what is it referring to:

`where they can be compared using a similarity measure.` -> `where image and text embeddings can be compared using a similarity measure`